apps/ledger-live-desktop/.unimportedrc.json

```diff
@@ -19,10 +19,18 @@
     "node_modules/**",
     "src/generate-cryptoassets-md.test.ts",
     "src/generate-cryptoassets-md.ts",
+
     "src/newArch/Collectibles/**",
-    "src/newArch/WalletSync/**"
+    "src/newArch/WalletSync/**",
+    "src/newArch/components/ContextMenu/**"
```

Comment:
Needed since contextMenu is used inside Collectibles folder which is not used for the moment